.idea/codestyles/Project.xml

```diff
@@ -159,4 +159,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```

Comment:
Can you revert this?

Reply:
https://github.com/stripe/stripe-android/pull/7513